locale/pt_BR/default.xml

```diff
@@ -42,13 +42,13 @@ Siga o seguinte protocolo para fazer revisões eletrônicas em seu manuscrito:
 &nbsp; Para cada sugestão rejeitada, insira o texto original novamente e aplique <strong>negrito</strong>.
 
 <strong>Realizando Inserções e Remoções</strong>
-&nbsp; Indique inserções novas em <strong>negrito</strong>.
-&nbsp; Substitua seções removidas por: <strong>[texto removido]</strong>.
-&nbsp; Caso remova uma frase ou mais, indique com um aviso (ex.: <strong>[removidas 2 frases]</strong>).
+ Indique inserções novas em <strong>negrito</strong>.
+ Substitua seções removidas por: <strong>[texto removido]</strong>.
+ Caso remova uma frase ou mais, indique com um aviso (ex.: <strong>[removidas 2 frases]</strong>).
```

Comment:
These are not very important, but the &nbsp; are included in the en_US version and I'd rather have the same everywhere...